### PR TITLE
[New Rule] Image File Execution Options Injection

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/549

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
image

File: rules/windows/persistence_evasion_registry_ifeo_injection.toml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+creation_date = "2020/11/17"
+maturity = "production"
+updated_date = "2020/11/17"
+
+[rule]
+author = ["Elastic"]
+description = """
+The Debugger and SilentProcessExit registry key allows an attacker to launch intercept the execution of files, causing
+an a different process to be executed. This functionality is used by attackers and often targets common programs to
+establish persistence.
```
Comment (bm11100) [inline rules/windows/persistence_evasion_registry_ifeo_injection.toml:11]:
```suggestion
The Debugger and SilentProcessExit registry keys can allow an adversary to intercept the execution of files, causing a different process to be executed. This functionality can be abused by an adversary to
establish persistence.
```

Comment (bm11100) [reply rules/windows/persistence_evasion_registry_ifeo_injection.toml:11]: re-lint

File: rules/windows/persistence_evasion_registry_ifeo_injection.toml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+references = [
+    "https://oddvar.moe/2018/04/10/persistence-using-globalflags-in-image-file-execution-options-hidden-from-autoruns-exe/",
+]
+risk_score = 41
+rule_id = "6839c821-011d-43bd-bd5b-acff00257226"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
```
Comment (bm11100) [inline rules/windows/persistence_evasion_registry_ifeo_injection.toml:26]: verified logic on master build

**Review by bm11100 — APPROVED** [review]

**Review by devonakerr — APPROVED** [review]

**Pull request merged**